src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -78,6 +79,13 @@ class Blueprint
      */
     public $after;
 
+    /**
+     * The blueprint state instance.
+     *
+     * @var \Illuminate\Database\Schema\BlueprintState
```

Comment:
you may add the `null` here:

@var \Illuminate\Database\Schema\BlueprintState|null